app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -286,6 +286,11 @@ class SampleView extends React.Component {
           visible={this.state.sampleDetailsSidebarVisible}
           onClose={this.toggleSampleDetailsSidebar}
           sampleId={sample.id}
+          pipelineVersion={
+            reportDetails &&
```

Comment:
You can also do `get('pipeline_info.pipeline_version', reportDetails)` and `import { get } from "lodash/fp"`. Slightly cleaner.

Reply:
`get` will return undefined if any part of the chain fails.

Reply:
thanks that's nice